tests/commons/src/main/java/org/codice/ddf/admin/comp/graphql/GraphQlHelper.java

```diff
@@ -35,6 +36,11 @@
   @SuppressWarnings("squid:S2068" /* Password used internally */)
   public static final String PASSWORD = "admin";
 
+  private final Gson gsonPrettyPrint =
+      new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
+
+  private final Gson gson = new GsonBuilder().create();
```

Comment:
❓ Should these also be static too? The one in the file above is. Or should that one not be static?

Reply:
That's a good point, originally I was only making the one above static because it had to be, but for consistency I could make all of them static 🤔